e2e/cron_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("E2E TEST:cron", func() {
+	Context("E2E TEST:Validating cron test", func() {
+		It("Should return valid test results for cron", func() {
+			fileName := fmt.Sprintf("%02d-%02d-%d.json",
```

Comment:
%04d instead of %d to be more exact with year :)

Reply:
Using the existing format to be consistent https://github.com/ossf/scorecard/blob/main/cron/main.go#L57